### ISSUE TITLE
New Vet Center Facility: Portsmouth Vet Center Outstation

### ISSUE THREAD

**Issue by stefaniefgray** [department-of-veterans-affairs/va.gov-cms#13618]
## Intake
- [ ] What triggered this runbook? (Flag in CMS, Help desk ticket, Product team, VHA Digital Media)
Trigger: Flagged content view

- [ ] Link to associated help desk ticket (if applicable)
Help desk ticket: https://va-gov.atlassian.net/browse/VAHELP-5571

- [ ] Name of submitter (if applicable)
Submitter: @stefaniefgray 

- [ ] If the submitter is an editor, send them links to any relevate KB articles for the Vet Center product.
KB articles: <insert_kb_article_links>

- [ ] Link to facility in production:
Facility CMS link: https://prod.cms.va.gov/manchester-vet-center/portsmouth-vet-center-outstation
Facility API ID: vc_01081OS

### New Vet Center
[@TODO: KB ARTICLE FOR ADDING VET CENTERS - SEE runbook-vamc-facility-new]

#### CMS help desk steps
**Note: If the help desk is waiting on information from the facility staff or editor, add the "Awaiting editor" flag to the facility with a log message that includes a link to this ticket. Remove the flag when the ticket is ready to be worked by the Facilities team. Be sure to preserve the current moderation state of the node when adding or removing the flag.**
- [x] 1. Become aware that the new facility is now on the Facility API (typically, via a Flag).
- [x] 2. Check with RCS(?) what district it belongs to.
- [x] 3. Move the section to the appropriate district.
- [ ] 4. Communicate with editor (do they need to be onboarded)

[@TODO write sample email - SEE runbook-vamc-facility-new]

- [ ] 5. Create a [URL change request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&template=runbook-facility-url-change.md&title=URL+Change+for%3A+%3Cinsert+facility+name%3E). (**Note: The URL change request ticket blocks the completion of this ticket.**)

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13617

(Redirects deploy weekly on Wed. at 10am ET, or by requesting OOB deploy (of the revproxy job to prod) in #vfs-platform-support. Coordinate the items below and canonical URL change after URL change ticket is merged, deployed, and verified in prod.)

- [ ] 6. When editor has prepared content and let help desk know, reassign this issue to appropriate CMS engineer on Product Support team, for bulk publishing.

#### CMS engineer steps
- [ ] 7. Execute the steps of the URL change request ticket from step 5.
- [ ] 8. Bulk publish the nodes and facility.
- [ ] 9. Edits facility node by removing the `New facility` flag and saves the node.
- [ ] 10. Let help desk know this has been done.

#### CMS Help desk (wrap up)
- [ ] 11. Notify editor and any other stakeholders.

**Comment by stefaniefgray** [comment]
Closing, have removed new Vet Center flag

Turns out it was a duplicate of the archived Manchester Vet Center Outstation, same address

Will need to contact editor about getting it removed from VAST
